localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -575,6 +575,26 @@ def SNS_Topic_get_cfn_attribute(self, attribute_name):
     SNS_Topic_get_cfn_attribute_orig = sns_models.Topic.get_cfn_attribute
     sns_models.Topic.get_cfn_attribute = SNS_Topic_get_cfn_attribute
 
+    # Patch ES get_cfn_attribute(..) method
+    def ES_get_cfn_attribute(self, attribute_name):
+        if attribute_name in ['Arn', 'DomainArn']:
+            return aws_stack.es_domain_arn(self.params.get('DomainName'))
+        if attribute_name == 'DomainEndpoint':
+            es_details = aws_stack.connect_to_service('es').describe_elasticsearch_domain(
+                DomainName=self.params.get('DomainName'))
+            return es_details['DomainStatus']['Endpoint']
+        raise UnformattedGetAttTemplateException()
+
+    service_models.ElasticsearchDomain.get_cfn_attribute = ES_get_cfn_attribute
+
+    # Patch Firehose get_cfn_attribute(..) method
+    def Firehose_get_cfn_attribute(self, attribute_name):
+        if attribute_name == 'Arn':
+            return aws_stack.firehose_stream_arn(self.params.get('DeliveryStreamName'))
```

Comment:
Same as above - we could try to cache the value here, for performance reasons.

Reply:
Are you sure about this one @whummer? I believe `firehose_stream_arn` doesn't make any hard work.

Reply:
Good point, agreed. 👍